Pranjali/HTML/Assi6/assi6.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <style>
+            .a {
+                font-family: Arial, Helvetica, sans-serif;
+                background-color: rgb(137, 167, 224);
+                text-align: right;
+            }
+
+            .b {
+                background-color: rgb(137, 167, 224);
+                text-align: right;
+            }
+
+            #corners {
+                border-bottom-left-radius: 8px;
+                border-top-right-radius: 8px;
+            }
+        </style>
+    </head>
+    <body>
+        <h2 class="a" id="corners">MI-9 Intern HTML Assesment</h2>
+        <br />
+        <hr />
+        <h3 class="b" id="corners"><em>Personal Details</em></h3>
+        <p>
+            <label for="firstname"><strong>First Name:</strong></label>
+            <input type="text" id="firstname" name="firstname" placeholder="First Name" />
```

Comment:
palceholders second words 1st letter should be small like First name make this change everywhere